staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -369,7 +369,7 @@ func (s *Scheme) AddValidationFunc(srcType Object, fn func(ctx context.Context,
 // Validate validates the provided Object according to the generated declarative validation code.
 // WARNING: This does not validate all objects!  The handwritten validation code in validation.go
 // is not run when this is called.  Only the generated zz_generated.validations.go validation code is run.
-func (s *Scheme) Validate(ctx context.Context, options sets.Set[string], object Object, subresources ...string) field.ErrorList {
+func (s *Scheme) Validate(ctx context.Context, options []string, object Object, subresources ...string) field.ErrorList {
```

Comment:
Do we want to say anything about sortedness?  If it is sorted we can do binary search, but my guess is that we have so few options that it's not really worthwhile?